library/src/main/java/com/fstyle/library/DialogBase.java

```diff
@@ -0,0 +1,67 @@
+package com.fstyle.library;
+
+import android.app.Dialog;
+import android.content.Context;
+import android.content.DialogInterface;
+import android.support.annotation.NonNull;
+import android.view.View;
+import android.view.ViewGroup;
+import com.fstyle.library.internal.MDRootLayout;
+
+/** @author Aidan Follestad (afollestad) */
+class DialogBase extends Dialog implements DialogInterface.OnShowListener {
+
+  protected MDRootLayout view;
+  private OnShowListener showListener;
```

Comment:
ERROR: 'member def modifier' have incorrect indentation level 2, expected level should be 4.